armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -1966,6 +1969,16 @@ def buildEqRingSchedule(self, ringSchedule):
         locationSchedule : list
 
         """
+
+        def distanceFromOrigin(a):
+            origin = numpy.array([0.0, 0.0, 0.0])
+            p = numpy.array(a.spatialLocator.getLocalCoordinates())
+            return math.sqrt(((p - origin) ** 2).sum())
```

Comment:
can almost definitely skip the sqrt and still be proportional to distance from origin. need new name then though.